tar/multitape/multitape_delete.c

```diff
@@ -76,6 +77,7 @@ deletetape(TAPE_D * d, uint64_t machinenum, const char * cachedir,
 	int lockfd;
 	uint8_t lastseq[32];
 	uint8_t seqnum[32];
+	int rc = -1;	/* Assume an error will occur; will change if needed. */
```

Comment:
Hmm, I think I understand.  How about "Assume error exit is not due to archive-not-found" here...